codalab/lib/bundle_cli.py

```diff
@@ -857,21 +867,28 @@ def print_version(self):
             'Show usage information for commands.',
             '  help           : Show brief description for all commands.',
             '  help -v        : Show full usage information for all commands.',
+            '  help -v -d     : Show full usage information for all commands in the Markdown format.',
             '  help <command> : Show full usage information for <command>.',
         ],
         arguments=(
             Commands.Argument('command', help='name of command to look up', nargs='?'),
             Commands.Argument(
                 '-v', '--verbose', action='store_true', help='Display all options of all commands.'
             ),
+            Commands.Argument(
+                '-d',
+                '--docs',
```

Comment:
`--markdown`

Reply:
So short form would probably be `-m` then